content/docs/reference/internals.md

```diff
@@ -60,7 +60,7 @@ The return of this function contains each ``MatchData``, which will tell the tra
 
 ### Operators
 
-Operators are stored in ``github.com/coraza-waf/coraza/v2/operators`` and contains an initializer and an evaluation function. Initializers are used to apply arguments during compilation, for example, ``"@rx /\d+/"`` will run ``op.Init("/\\d+")``. ``op.Evaluate(tx, "args")`` is applied for each compiled variable and will return if the condition matches. Operators uses ``Transaction`` to create logs, capture fields and access additional variables from the transaction.
+Operators are stored in ``github.com/coraza-waf/coraza/v2/operators`` and contain an initializer and an evaluation function. Initializers are used to apply arguments during compilation, for example, ``"@rx /\d+/"`` will run ``op.Init("/\\d+")``. ``op.Evaluate(tx, "args")`` is applied for each compiled variable and will return if the condition matches. Operators uses ``Transaction`` to create logs, capture fields and access additional variables from the transaction.
```

Comment:
```suggestion
Operators are stored in ``github.com/corazawaf/coraza/tree/v3/dev/internal/operators`` and contain an initializer and an evaluation function. Initializers are used to apply arguments during compilation, for example, ``"@rx /\d+/"`` will run ``op.Init("/\\d+")``. ``op.Evaluate(tx, "args")`` is applied for each compiled variable and will return if the condition matches. Operators uses ``Transaction`` to create logs, capture fields and access additional variables from the transaction.
```

This link is outdated, but not even sure it is worth updating it with a link to the current `v3/dev` branch. It might be better to wait for the v3 tag and update the docs after it

Reply:
I am not familiar with the dev workflow of the project yet, but I think we can keep it up to date if possible 👍